### PR TITLE
add a loading state to open in stream button

### PR DESCRIPTION
**Pull request by andygnewman**
loading

### CODE REVIEW

File: components/card-open-in-stream/_main.scss [hunk 1]
```diff
@@ -95,21 +95,22 @@
...
-	cursor: pointer;
-	&:after {
-		@include nextIcon(arrow-down, getColor('grey-tint5'), 10);
-		content: '';
-		position: relative;
-		top: 1px;
-		padding: 0 2px;
-	}
-	.is-open & {
+	&.is-static {
```
Comment (i-like-robots) [inline components/card-open-in-stream/_main.scss]: We can just use `&[disabled]` or `&[aria-busy=true]`, they're both 'state' so no need for an extra flag =]

File: components/card-open-in-stream/inline-article/index.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ module.exports = React.createClass({
 		if (!this.state.articleExpanded) {
 			this.setState({
 				articleLoading: true,
+				buttonLabel: 'Loading',
```
Comment (i-like-robots) [inline components/card-open-in-stream/inline-article/index.js]: Do we want an ellipsis char (`…`) at the end to create a heightened sense of suspense?

Comment (andygnewman) [inline components/card-open-in-stream/inline-article/index.js]: Not a fan, unless they're animated, though happy for a design view. Not sure we have anything similar elsewhere?

File: components/card-open-in-stream/_main.scss (hunk 1 above)
Comment (andygnewman) [inline components/card-open-in-stream/_main.scss]: Good point, will change it.

**Comment by i-like-robots** [comment]
Awesome :ship: (might invert the styles at some point as the button isn't disabled/busy by default)

**Pull request merged**